compiler/tests-spec/testData/diagnostics/notLinked/contracts/analysis/smartcasts/neg/6.fir.kt

```diff
@@ -111,7 +111,7 @@ import contracts.*
 fun case_1(value_1: Any?, value_2: Int?) {
     value_1.case_1(value_2)
     <!AMBIGUITY!>println<!>(value_1.<!UNRESOLVED_REFERENCE!>length<!>)
-    println(value_2.<!INAPPLICABLE_CANDIDATE!>inv<!>())
+    println(value_2.<!UNSAFE_CALL!>inv<!>())
```

Comment:
The contract for the extension function `case_1` above is that, if returned, the given argument is not null. Thus, `value_2` should be not null here.

Many other unsafe calls in this file are originated from such incorrect contract propagations.